examples/billing/remove_billing_setup.rb

```diff
@@ -26,12 +26,16 @@ def remove_billing_setup(customer_id, billing_setup_id)
   # ENV['HOME']/google_ads_config.rb when called without parameters
   client = Google::Ads::GoogleAds::GoogleAdsClient.new
 
-  billing_setup_service = client.service(:BillingSetup)
+  billing_setup_service = client.service.billing_setup
 
   resource =  client.path.billing_setup(customer_id, billing_setup_id)
+  op = client.operation.billing_setup
+  op["remove"] = resource
```

Comment:
We should be able to do `client.operation.remove_resource.billing_setup(resource)`